aws/aws-how-to/security/ubuntu-security-on-aws.rst

```diff
@@ -0,0 +1,47 @@
+Ubuntu Security on AWS
+######################
+
+Ubuntu security features
+************************
+
+Ubuntu on AWS provides all security features available to Ubuntu Server. For more detailed information about how Ubuntu offer security, refer to the following `Introduction to Security <https://documentation.ubuntu.com/server/explanation/intro-to/security/>`_.
+
+
+AWS security features
+*********************
+
+AWS offers comprehensive security and data protection in the cloud. `Security in Amazon EC2 <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security.html>`_ explains how user can benefit from AWS security features.
+
+Secure Boot and TPM
+~~~~~~~~~~~~~~~~~~~
+
+AWS Secure Boot is a feature providing secure boot process of Amazon EC2 so instance only boots trusted softwares. To create and configure secure boot image with Ubuntu AMI, check out :doc:`./use-secureboot-and-vtpm`.
+
+CIS hardened image
+~~~~~~~~~~~~~~~~~~
+
+CIS hardened images are built with configuration by the Center for Internet Security (CIS) for use on Amazon EC2.
+
+To create hardened image with AWS Ubuntu Pro image, follow instructions in `Comply with CIS or DISA STIG on Ubuntu 20.04 with Ubuntu Security Guide <https://ubuntu.com/tutorials/comply-with-cis-or-disa-stig-on-ubuntu#1-overview>`_.
```

Comment:
```suggestion
To create a hardened image using Ubuntu Pro, follow the instructions in `Comply with CIS or DISA STIG on Ubuntu 20.04 with Ubuntu Security Guide <https://ubuntu.com/tutorials/comply-with-cis-or-disa-stig-on-ubuntu#1-overview>`_.
```
The instructions mentioned in that link only seem to [perform an audit](https://ubuntu.com/tutorials/comply-with-cis-or-disa-stig-on-ubuntu#5-run-the-ubuntu-security-guide), and don't really tell you how to harden the image. So we should either change the link to a more appropriate one, or change our text and include details for hardening. Some useful links:
- https://ubuntu.com/security/certifications/docs/usg/cis/compliance
- the video at https://ubuntu.com/security/cis

Reply:
I searched a little bit and Carlos presented this long time ago with Ubuntu Pro image so we can directly link to this video for tutorial.